src/main/steps/privacy-policy/content.ts

```diff
@@ -1,7 +1,7 @@
 import { TranslationFn } from '../../app/controller/GetController';
 
 export const en = {
-  title: 'The Adoption Service Privacy policy',
+  title: 'The  Criminal Injuries Compensation Tribunal Privacy policy',
```

Comment:
Minor comment: There is a typo. Additional space can be removed from 'The  Criminal'.